packages/core/src/lib/wallet-selector.types.ts

```diff
@@ -14,6 +14,7 @@ export type WalletSelectorStore = ReadOnlyStore;
 
 export type WalletSelectorEvents = {
   networkChanged: { walletId: string; networkId: string };
+  uriChanged: { walletId: string; uri: string };
```

Comment:
We will need to document this new event `uriChanged` in the docs of the selector in a similar way we do it for `networkChanged` event: this can be done in this file/section:
https://github.com/near/wallet-selector/blob/SQC-288/qr-code-in-ui/packages/core/docs/api/selector.md#onevent-callback